src/features/deposit/components/DepositWidget.tsx

```diff
@@ -17,23 +18,22 @@ import {
   type DepositFormNearValues,
 } from "./Form/DepositFormNear"
 
-type DepositWidgetProps = {
-  accountId: string
-  signAndSendTransactionsNear: (transactions: Transaction[]) => void
-}
-
```

Comment:
I strong believer that Props should live along with a component itself.

Reply:
No, because this is the root component, which has a type exported from the SDK, and it is the same as for the swap, so then we should take into account it also.

Reply:
Yes, all components should have props along with them and not separated. If a package wants to export it, then it should re-export it.